rasa/nlu/training_data/message.py

```diff
@@ -11,6 +11,11 @@
 from rasa.nlu.utils import ordered
 
 
+INTENT_RANKING_KEY = "intent_ranking"
```

Comment:
let's move it to nlu/constants.py
```suggestion
INTENT_RANKING = "intent_ranking"
```

Reply:
I'll move it. I'd prefer to keep `_KEY` part because it tells me exactly what the variable is doing